### DISCUSSION TITLE
White background Graph

### DISCUSSION THREAD

**Discussion by svenvg93** [oetiker/SmokePing#378] [Q&A] [answered]
I would like to have graphs with a white background instead of the greyish . This way it fits better in our monthly reporting. 
Just like how it is on the frontpage of the website; 
![image](https://github.com/oetiker/SmokePing/assets/4511676/14441d7a-6f95-433c-b108-0d825ffd6c78)

I cant seem to find the option to do this in the config files. @oetiker can you point me in the right direction?

**Comment by oetiker** [comment, marked as answer]
the documentation should be updated on the website ....

<h2 id="Presentation-mandatory-section">*** Presentation *** <i>(mandatory section)</i></h2>

<p>Defines how the SmokePing data should be presented.</p>

<p>The following variables can be set in this section:</p>

<dl>

<dt id="template-mandatory-setting"><b>template</b> <i>(mandatory setting)</i></dt>
<dd>

<p>The webpage template must contain keywords of the form <b>&lt;##</b><i>keyword</i><b>##&gt;</b>. There is a sample template included with Smoke
Ping; use it as the basis for your experiments. Default template contains a pointer to the SmokePing counter and homepage. I would be glad if you 
would not remove this as it gives me an indication as to how widely used the tool is.</p>

</dd>
<dt id="charset"><b>charset</b></dt>
<dd>

<p>By default, SmokePing assumes the &#39;utf-8&#39; character set. If you use something else, this is the place to speak up.</p>

</dd>
<dt id="htmltitle"><b>htmltitle</b></dt>
<dd>

<p>By default, SmokePing will render the title of the graph in the image, when set to &#39;yes&#39; the title is inserted in the html page.</p>

</dd>
<dt id="graphborders"><b>graphborders</b></dt>
<dd>

<p>By default, SmokePing will render gray border on a light gray background, if set to &#39;no&#39; borders will be hidden and the background and 
canvas will be transparent.</p>

</dd>
<dt id="colortext"><b>colortext</b></dt>
<dd>

</dd>
<dt id="colorbackground"><b>colorbackground</b></dt>
<dd>

<p>By default, SmokePing will render the background light gray, which may be overridden here with your own RGB value</p>

</dd>
<dt id="colorborder"><b>colorborder</b></dt>
<dd>

<p>By default, SmokePing will render the border gray, which may be overridden here with your own RGB value</p>

</dd>
</dl>

<p>The following sections are valid on level 1:</p>

<dl>

<dt id="overview-mandatory-section"><b>+overview</b> <i>(mandatory section)</i></dt>
<dd>
